build.sbt

```diff
@@ -105,6 +105,10 @@ lazy val root = (project in file("."))
     Docker / version            := (ThisBuild / version).value.replaceAll("-SNAPSHOT", "-latest").toLowerCase,
     Docker / packageName        := s"${name.value}",
     Docker / dockerExposedPorts := Seq(8080),
+    Docker / dockerAliases      := Seq(
+      dockerAlias.value,
+      Option(System.getenv("COMMIT_SHA")).fold(dockerAlias.value)(t => dockerAlias.value.withTag(Some(t)))
```

Comment:
@micdes-pagopa so this is injected by the CI, right?

Reply:
Yes, I will set `COMMIT_SHA ` from GH actions